python/ray/tune/utils/release_test_util.py

```diff
@@ -136,38 +140,8 @@ def timed_tune_run(
 
     _train = function_trainable
 
-    aws_key_id = os.getenv("AWS_ACCESS_KEY_ID", "")
-    aws_secret = os.getenv("AWS_SECRET_ACCESS_KEY", "")
-    aws_session = os.getenv("AWS_SESSION_TOKEN", "")
-
     if durable:
-
-        class AwsDurableTrainable(TestDurableTrainable):
-            AWS_ACCESS_KEY_ID = aws_key_id
-            AWS_SECRET_ACCESS_KEY = aws_secret
-            AWS_SESSION_TOKEN = aws_session
```

Comment:
This stuff is not needed anymore due to propagating the env vars in the runtime env from the previous PR?

Reply:
That's one thing - the other is that the cluster infrastructure now correctly sets up credentials on all worker nodes